tests/generic/bug01382-006.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Test for PHP-1382: Last document missing
```

Comment:
Can you add "(with getmore)" to the title? At a glance, this looks very similar to 002.